man/systemd.netdev.xml

```diff
@@ -911,6 +916,23 @@
           <para>Accepts the same key as <literal>[FooOverUDP]</literal></para>
         </listitem>
       </varlistentry>
+      <varlistentry>
+        <term><varname>Sequence=</varname></term>
+        <listitem>
+          <para>Takes a boolean value. If set to yes, then packets are serialized. Only applies for ERSPAN tunnel.
+          Defaults to <literal>unset</literal>.
+          </para>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term><varname>ErspanIndex=</varname></term>
```

Comment:
We keep the capitalization of abbreviations when concatenating multiple words together. So this should be "ERSPANIndex".